packages/neon/neon_talk/lib/l10n/en.arb

```diff
@@ -12,5 +12,10 @@
       "type": {}
     }
   },
-  "roomSendMessage": "Send a message..."
+  "roomSendMessage": "Send a message...",
```

Comment:
```suggestion
  "roomSendMessage": "Write a message...",
```
Let's get really nit picky :sweat_smile: 
Typing here does not send it yet. 
This also aligns it with the upstream string.